components/org.wso2.carbon.identity.api.server.input.validation/org.wso2.carbon.identity.api.server.input.validation.v1/src/main/resources/inputValidation.yaml

```diff
@@ -79,6 +79,68 @@ paths:
           $ref: '#/components/responses/Forbidden'
         '500':
           $ref: '#/components/responses/ServerError'
+  /validation-rules/{field}:
```

Comment:
Are we going to keep the existing APIIs as well?

Reply:
Yes